websites/T/TruckersMP/dist/metadata.json

```
@@ -21,24 +21,30 @@
     "stats.truckersmp.com",
     "map.truckersmp.com"
   ],
-  "version": "2.0.2",
+  "version": "3.0.0",
```

That's not how you semver
```suggestion
  "version": "2.0.3",
```